motionParam.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import math
+import matplotlib as plt
```

Comment:
I noticed that you do  `import matplotlib as plt`, what you actually want is `import matplotlib.pyplot as plt`